modules/ssh/manifests/service.pp

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+class ssh::service {
+    case $::operatingsystem {
+        Darwin : {
+            exec {
```

Comment:
-w is no longer valid in Mojave.  Since ssh is enabled in the deploystudio workflow, we don't need to use an exec here.  Simply use the service resource.